src/Label/LabelControllerProvider.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @file
+ */
+
+namespace CultuurNet\UDB3\Silex\Label;
+
+use CultuurNet\UDB3\Symfony\Offer\EditOfferRestController;
+use Silex\Application;
+use Silex\ControllerCollection;
+use Silex\ControllerProviderInterface;
+
+class LabelControllerProvider implements ControllerProviderInterface
```

Comment:
Ik denk dat we deze best hernoemen naar OfferControllerProvider, omdat hier later ook de translations bijkomen (en later nog andere zaken ook hoogstwaarschijnlijk). Of we moeten deze file weg laten en ze in EventControllerProvider en PlaceControllerProvider steken, dat gaat ook.

We kunnen alleszins moeilijk later ook nog eens een TranslationControllerProvider voorzien (naast LabelControllerProvider), want dan moeten we weer opnieuw 2 instances van EditOfferRestController gaan voorzien. (Die nu event_label_controller en place_label_controller heten)